lua/mason-registry/commitlint/init.lua

```diff
@@ -0,0 +1,17 @@
+local Pkg = require "mason-core.package"
+local _ = require "mason-core.functional"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "commitlint",
+    desc = "commitlint checks if your commit messages meet the conventional commit format.",
+    homepage = "https://commitlint.js.org/",
+    languages = { Pkg.Lang.GitCommit },
+    categories = { Pkg.Cat.Linter },
+    install = npm.packages {
+        "@commitlint/cli",
+        "@commitlint/config-conventional",
+        "commitlint-format-json",
```

Comment:
Is this to enable linter integrations?

Reply:
Yes. It's supported by null-ls:

https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/doc/BUILTINS.md#commitlint

And all 3 of those packages are required for it to work.

Reply:
Ok, cool!